### DISCUSSION TITLE
asciidocfx preview not showing css

### DISCUSSION THREAD

**Discussion by Johan1974** [asciidocfx/AsciidocFX#531] [Q&A]
Hello,

Just started using Ascciidocfx and i notice the preview is not showing my document with css applied. I know my code is correct as it does work with visual studio. I am not seeing any comments about it in earlier discussions here hopefuly someone can explain this to me. Do i need to see some option to listen to my own custom css and not some default css hidden in asciidocfx? 

Thanks!

Johan

**Comment by rahmanusta** [comment]
Hi, I can check if you share some document snippet

**Comment by Johan1974** [comment]
Hello Rahman,

I have the followoing in my root

index.adoc with the following lines

--- >>  include::variables.adoc[]

within variables.adoc (also in my root)  i have the following line

--- >> :stylesheet     :   css\styles.css

This same project loaded in VSCode is working fine with the css

This is a fresh install of asciidofx so maybe i am missing some initial
settings that needs to be set?











Met vriendelijke groet / With Kind regards,

Johan Lijffijt

**Comment by rahmanusta** [comment]
Hi, sorry something wrong with this. It will apply the css, if you generate an html file or if you override default css in settings > Location settings.

**Comment by Johan1974** [comment]
Hello,

I noticed it is working now when generating a html. But i would like to see
this also in the preview.
Otherwise i have to generate a htm; for every change  made and then maybe
again if i made a mistake.


Met vriendelijke groet / With Kind regards,

Johan Lijffijt
